_faqs/faq-12.md

```diff
@@ -0,0 +1,8 @@
+---
+question: I want to cite RIOT. Which reference should I use?
+---
+
+Please consider the following references for citation.
+
+  * Emmanuel Baccelli, Cenk G&uuml;ndogan, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdf/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
```

Comment:
New to the website so asking the stupid question: Do we really have to escape things down to ASCII? (If so, I think it's G&uuml;ndo&gbreve;an; if not, Gündoğan).

Examples of non-ASCII characters in existing (and working) text are _data/riot_boards.yml (TRÅDFRI) and the various names in community.html.

(There's nothing wrong per se with it in this case, I'm more trying not to get us take up habits that others later imitate and needlessly overencode.)

Reply:
thanks for spotting. copy&paste failure. fixed.

> Do we really have to escape things down to ASCII? 
>
we don't need to but we allow.